integration-tests/opentelemetry-vertx/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.native.additional-build-args =-H:ReflectionConfigurationFiles=reflection-config.json
```

Comment:
Is this only needed because we're testing/checking span values?

Reply:
Yes. This is in the integration test modules, so not a big deal.

Reply:
Well. I just wanted to make sure that a user would not have to do this in their app. ;)

Reply:
It should have worked it `@RegisterForReflection`, but for some reason it didn't work. I'll have to look into that.